README.md

```diff
@@ -43,9 +43,25 @@ spec:
   serviceNetwork:
   - 172.30.0.0/16
 ```
+Alternatively, OVN can be configured as follows:
+```yaml
+apiVersion: config.openshift.io/v1
+kind: Network
+metadata:
+  name: cluster
+spec:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  networkType: OVNKubernetes
+  serviceNetwork:
+  - 172.30.0.0/16
+```
 
 *Corresponding Operator Config*
 This configuration is the auto-generated translation of the above Cluster configuration.
+Note: the hostPrefix is converted into hostSubnetLength (hostSubnetLength = 32 - hostPrefix).
+When OVNKubernetes is used it is converted back to hostPrefix format and appended to the cidr.
```

Comment:
Converting the configuration from the generic form into the form needed by the network plugins is what the CNO does. You don't need to call out specific examples. (Likewise with the similar hostSubnetLength comment a bit later.)

Reply:
No, `hostSubnetLength` is *not* wrong here. We're not talking about OpenShift SDN's configuration, we're talking about the cluster network operator's configuration, and the cluster network operator's configuration uses a `hostSubnetLength` field, regardless of what plugin type you are using.